contracts/dtoken/src/lib.rs

```diff
@@ -68,6 +74,41 @@ impl Dtoken {
     pub fn get_underlying_balance() -> u128 {
         BigDecimal::from_str("1.2").unwrap().to_u128().unwrap()
     }
+
+    pub fn internal_unwrap_balance_of(&self, account_id: &AccountId) -> Balance {
```

Comment:
@TymofiiSavinov do we need to expose this function? Maybe it should be something like `ft_balance_of`?

P.S.
The same for all `internal_*` functions should any of the be exposed to public?